src/queries/FMI/getBuoysQuery.ts

```diff
@@ -6,14 +6,16 @@ const query = '&request=getFeature&storedquery_id=fmi::ef::stations&networkid=13
 export async function getBuoys() {
   const sites: Site[] = [];
   const response = await GetRawXMLResponse(query);
-  const facilities = response.getElementsByTagName('ef:EnvironmentalMonitoringFacility');
-  for (const facility of facilities) {
-    const id = facility.getElementsByTagName('gml:identifier')[0].firstChild!.nodeValue!;
-    const name = facility.getElementsByTagName('ef:name')[0].firstChild!.nodeValue!;
-    // tslint:disable-next-line:max-line-length
-    const pos = facility.getElementsByTagName('ef:representativePoint')[0].firstElementChild!.firstElementChild!.textContent!.split(' ');
+  if (response) {
```

Comment:
Tämmösessä keississä alkaa olla nätimpää jo tehdä käänteisenä, eli
```
if (!response){
    return sites;
}
const facilities = ...
```

Reply:
Filosofointia esim. https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement